packages/estatico-boilerplate/src/assets/js/helpers/polyfills.js

```diff
@@ -6,12 +6,24 @@
  *
  * If a polyfill is expected to be used only in exceptional cases it could make sense to load it
  * where needed instead of here
+ *
+ * !IMPORTANT! Check with core-js to see which polyfills are included for you automatically with @babel/preset-env
```

Comment:
@christiansany, the linter isn't happy about the length of this line. Apart from that: Perfect!